plugins/woocommerce/templates/emails/plain/customer-on-hold-order.php

```diff
@@ -23,7 +23,11 @@
 
 /* translators: %s: Customer first name */
 echo sprintf( esc_html__( 'Hi %s,', 'woocommerce' ), esc_html( $order->get_billing_first_name() ) ) . "\n\n";
-echo esc_html__( 'Thanks for your order. It’s on-hold until we confirm that payment has been received. In the meantime, here’s a reminder of what you ordered:', 'woocommerce' ) . "\n\n";
+echo esc_html__( 'Thanks for your order. It’s on-hold until we confirm that payment has been received.', 'woocommerce' ) . "\n\n";
```

Comment:
This sentence 

echo esc_html__( 'Thanks for your order. It’s on-hold until we confirm that payment has been received. In the meantime, here’s a reminder of what you ordered:', 'woocommerce' ) . "\n\n";

to

echo esc_html__( 'Thanks for your order. It’s on-hold until we confirm that payment has been received. ', 'woocommerce' ) . "\n\n";